### PR TITLE
chore: increase the speed of the size action in CI

### PR DESCRIPTION
**Pull request by kangju2000**
# Overview

Improve CI speed through the prepack script.

## PR Checklist

- [x] I did below actions if need

1. I read the [Contributing Guide](https://github.com/toss/suspensive/blob/main/CONTRIBUTING.md)
2. I added documents and tests.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -61,3 +62,4 @@ jobs:
         with:
           working-directory: ./websites/visualization
           comment-strategy: 'always'
```
Comment (kangju2000) [inline .github/workflows/ci.yml:64]: [`skip-insignificant`](https://github.com/transferwise/actions-next-bundle-analyzer?tab=readme-ov-file#usage) is skip commenting on PRs if there are no signficant changes in page sizes.

Do you think we should always show the bundle size of the visualization website, even if there's no change in size?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -61,3 +62,4 @@ jobs:
...
+          create-issue: false
```
Comment (kangju2000) [inline .github/workflows/ci.yml:65]: I'll make sure no issue is created. https://github.com/toss/suspensive/issues/1128

Comment (manudeli) [reply .github/workflows/ci.yml:65]: But if we need to diff default branch with Pull Request branch, don't we need to create issue?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -53,6 +53,7 @@ jobs:
       - if: matrix.command == 'prepack'
         uses: preactjs/compressed-size-action@v2
         with:
+          build-script: 'prepack'
```
Comment (manudeli) [inline .github/workflows/ci.yml:56]: 👍

File: .github/workflows/ci.yml (hunk 2 above)
Comment (kangju2000) [reply .github/workflows/ci.yml:65]: The issue only records the current bundle size of the website, so creating an issue isn't essential for the action to run properly.

If you want the bundle size to always be updated through issues, we can keep generating the issues!

What do you think?

Comment (manudeli) [reply .github/workflows/ci.yml:65]: > If you want the bundle size to always be updated through issues

I don't want to make issue too. Thanks!

**Review by manudeli — APPROVED** [review]

**Pull request merged**